jme3-core/src/main/java/com/jme3/renderer/opengl/GLRenderer.java

```diff
@@ -2852,7 +2852,8 @@ public void clearVertexAttribs() {
         for (int i = 0; i < attribList.oldLen; i++) {
             int idx = attribList.oldList[i];
             gl.glDisableVertexAttribArray(idx);
-            if (context.boundAttribs[idx].get().isInstanced()) {
+            VertexBuffer buffer = context.boundAttribs[idx].get();
+            if (buffer != null && buffer.isInstanced()) {
```

Comment:
I have tested that for 15 mins, different worlds & different vehicle parameters, it works fine !

Reply:
Thank you!